string id
string properties
---
bool success
